src/main/java/com/firecontroller1847/truediscordlink/DiscordManager.java

```diff
@@ -496,6 +499,69 @@ private void modifyAddCheckMentions(AtomicReference<String> content, AtomicRefer
         }
     }
 
+    // Utility method to check for any channel mentions
+    private void modifyAddCheckChannelTags(AtomicReference<String> content, AtomicReference<ArrayList<String[]>> modifications, Player player) {
+        // Check if connected
+        if (!isBotConnected()) {
+            return;
+        }
+
+        // Check if Enabled
+        if (!discordlink.getConfig().getBoolean("tagging.enable_channel_tagging")) {
+            return;
+        }
+
+        // Check for user permission
+        if (player != null && !TrueDiscordLink.hasPermission(player, "truediscordlink.tagging")) {
```

Comment:
Let's separate the permissions from role and user tagging and channel tagging, since many people will probably want channel tagging enabled but role tagging disabled. Don't change the name of the user tagging for backward compatibility purposes, though.